storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```
@@ -1183,7 +1184,8 @@
       (.readBlobTo blob-store (ConfigUtils/masterStormConfKey storm-id) (FileOutputStream. (ConfigUtils/supervisorStormConfPath tmproot)) nil)
       (finally
         (.shutdown blob-store)))
-    (FileUtils/moveDirectory (File. tmproot) (File. stormroot))
+    (try (FileUtils/moveDirectory (File. tmproot) (File. stormroot)) (catch Exception e))
```

Why are we ignoring this exception now?